### DISCUSSION TITLE
✅ How can I disable the keyboard accessory view when using the iPad with the Magic Keyboard

### DISCUSSION THREAD

**Discussion by logicito** [isontheline/pro.webssh.net#1180] [General]
If I select the keyboard on top, it will disappear but then the Magic Keyboard will stop responding, is there a way to fully disable it, I don’t see the need when you have an external keyboard, thanks

**Comment by isontheline** [comment]
Hello @logicito 👋 

Inside the settings : 
iOS Settings > WebSSH > SSH > Show virtual keys above keyboard

**Reply by logicito** [reply]
Thank you, great application by the way

**Reply by tommythorn** [reply]
I don’t see that option on iPhone (when using a physically attached keyboard I wish to maximize terminal real-estate and ideally go 100% full screen)

**Reply by isontheline** [reply]
Hey @tommythorn 👋

This option has been dropped since WebSSH 30.X, I can open a new issue to introduce it again?

**Reply by tommythorn** [reply]
Thanks, but I'd actually like to suggest a slightly larger feature: the ability to go completely full screen with the only UI element a barely visible *translucent* "Open menu button" that when tapped give access to all the other options.

I don't so much care about the details, but the overriding point is that when we are engaging with WebSSH, most of the time we really want as much screen real-estate as possible for the terminal output on the tiny screen.  When a physical keyboard is attached, that would mean 100% full screen.  When the onscreen keyboard is used, the current bottom implementation strike me a great, but there is significant wasted space above.

What do you think of this idea?  Do you have a bounty scheme for us to support favorite features?

**Reply by isontheline** [reply]
I'm appending your great ideas to this issue : https://github.com/isontheline/pro.webssh.net/issues/645

No bounty scheme unfortunately

**Reply by isontheline** [reply]
I will work on a - true - fullscreen mode but can't promise any ETA

Just an update, you can hide both top and bottom bars inside the "..." menu : 
<img width="326" height="707" alt="image" src="https://github.com/user-attachments/assets/ea945b98-c841-4d07-8000-884b9d2c2275" />
